### PR TITLE
New Devices (Z-Wave Switch): Shelly Switches

### PR DESCRIPTION
**Pull request by lelandblue**
This is a PR Request for #Shelly/Qubino devices being added to the Z-Wave Switch Driver. 

There will be a follow on PR for additional devices that will be CxS. 

EDIT: I am limiting the scope of this PR to include only the devices that believed to use two endpoints with "2" in their device name.

### CODE REVIEW

**Review by greens — APPROVED** [review]

**Review by greens — APPROVED** [review]

File: drivers/SmartThings/zwave-switch/fingerprints.yml [hunk 1]
```diff
@@ -898,6 +898,61 @@ zwaveManufacturer:
...
+    manufacturerId: 0x0460
+    productId: 0x0081
+    productType: 0x0002
+    deviceProfileName: dual-metering-switch
+  - id: 1120/2/131
+    deviceLabel: Wave 1
+    manufacturerId: 0x0460
+    productId: 0x0083
+    productType: 0x0002
+    deviceProfileName: switch-binary-indicator
```
Comment (greens) [inline drivers/SmartThings/zwave-switch/fingerprints.yml:913]: the indicator preference here won't work unless you point out which configuration parameter is used here: https://github.com/SmartThingsCommunity/SmartThingsEdgeDrivers/blob/main/drivers/SmartThings/zwave-switch/src/preferences.lua#L310

**Review by greens — CHANGES_REQUESTED** [review]
the `switch-binary-indicator` profile needs to change

**Review by wkhenon — APPROVED** [review]

**Comment by lelandblue** [comment]
Hey @greens can I get a re-review on this PR please, it is ready to merge after that completes. Thank you!

**Review by greens — APPROVED** [review]

**Review by greens — CHANGES_REQUESTED** [review]
Ah, there are some test failures...

**Comment by greens** [comment]
working on them

**Review by greens — APPROVED** [review]
I think my changes look great.

**Pull request merged**